src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/VsInstanceTelemetryConsts.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Telemetry
+{
+    public abstract class VsInstanceTelemetryConsts
```

Comment:
Consider using dedicated telemetry classes rather than a catch all. 

See: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/RestoreTelemetryEvent.cs
See: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/RestoreTelemetryEvent.cs
See: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchTelemetryEvent.cs

Reply:
Ok. Added dedicated 3 telemetry event classes. Please check now. They're the ones actually emitted.
NuGetPowershellLoadedEvent.cs
NuGetPowershellVSInstanceCloseEvent.cs
NuGetPowershellVSSolutionCloseEvent.cs